core/types/transaction.go

```diff
@@ -78,14 +78,14 @@ type TxData interface {
 	accessList() AccessList
 	data() []byte
 	gas() uint64
-	gasPrice() *big.Int
+	// gasPrice() *big.Int //remove 581
```

Comment:
Please remove unused code. If we ever want this code back, we can dig it out of git history :)